docs/testnet11.md

```diff
@@ -0,0 +1,70 @@
+
+## Setup Instructions
+
+Testnet11 uses a dedicated P2P port (16311) so that nodes from the usual tesnet (10) do not automatically attempt connecting to it.
+
+We reiterate that only the included miner should be used to maintain a level playing field.
+
+First, we set-up a node:
+1. Download and extract the rusty-kaspa binaries <add URL>. Alternatively, you can compile it from source yourself by following the instructions [here](https://github.com/kaspanet/rusty-kaspa/blob/master/README.md). The rest of the instructions are written assuming the former option. If you choose to locally compile thecode, replace any command of the form ``<program> <arguments>`` with ``cargo run --bin <program> --release -- <arguments>`` (see example in the next item). All actions described below should be performed on a command line window where you navigated to the directory into which the binaries were extracted.
+2. Start the ``kaspad`` client with ``utxoindex`` enabled:
+
+```
+kaspad --testnet --netsuffix=11 --utxoindex
+```
+  If you complied the code yourself, you should instead run
+```
+cargo run --bin kaspad --release -- --testnet --netsuffix=11 --utxoindex
+```
+  Leave this window open, there is no need to touch it as long as the node is running. Closing it will stop the node.
+  
+If you want to transmit transactions, first create a Rothschild wallet
+1. Run ``rothschild`` to generate a wallet
+2. The output with provide you with a private key (that looks like a bunch of gibberish) and a public address (that looks like "kaspatest:" followed by a bunch of gibberish). For example, the output could look like this:
+     ```
+     2023-06-25 18:00:58.677+00:00 [INFO ] Connected to RPC
+     2023-06-25 18:00:58.677+00:00 [INFO ] Generated private key aa1c554386218eb28c4bsf6a02e5943799cf951dac7301324d88dec2d0119fce and address kaspatest:qzlpwt49f0useql6w0tzpnf8k2symdv5tu2x2pe9r9nvngw8mvx57q0tt9lr5. Send some funds to this address and rerun rothschild with `--private-key aa1c554386218eb28c4bsf6a02e5943799cf951dac7301324d88dec2d0119fce`
+      ```
+     Here, the private key is ```aa1c554386218eb28c4bsf6a02e5943799cf951dac7301324d88dec2d0119fce``` and the address is ```kaspatest:qzlpwt49f0useql6w0tzpnf8k2symdv5tu2x2pe9r9nvngw8mvx57q0tt9lr5```
+3. Put some money into the wallet. This could be done by either mining to that wallet (see below) or asking other participants to send money to your public address in the \#testnet Discord channel.
+4. Once the wallet has been funded, run Rothschild with the private key:
+   ```
+   rothschild --private-key <private-key> -t=50
+   ```
+  The last parameter ``-t=50`` means Rothschild will attempt broadcasting 50 transactions per second. We encourage participants to run with different TPS values. However, to prevent congestion, the code is hardwired not to go above 100 TPS.
```

Comment:
-> "However, in order to encourage transaction spread and to simulate organic usage we highly recommend not to go above 100 TPS"